CSC/ECE 517 Fall 2024 - E2488. Reimplement add TA

MakarandPundlik opened the pull request.

MakarandPundlik committed:
Updated README.md added contributors name

MakarandPundlik committed:
Added npm install command

Commit:
UI enhancements to the TA and Course page

MakarandPundlik committed:
resolved a bug by adding select TA to the final array and removed it from the interface

AnuragGorkar committed:
Merge pull request #5 from MakarandPundlik/makarand_features

resolved a bug by adding select TA to the final array and removed it …

MakarandPundlik committed:
Merge pull request #1 from MakarandPundlik/manage_ta_page

Manage TAs and Courses page

pnprathima commented on README.md:
```diff
@@ -44,3 +47,8 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
 To learn React, check out the [React documentation](https://reactjs.org/).
+
```
You can remove contibutors from here, since its a common readme file.

MakarandPundlik committed:
added unit test cases for columns

MakarandPundlik committed:
added unit test cases for coursecolumns

MakarandPundlik committed:
added test cases for column button

MakarandPundlik committed:
Merge pull request #6 from MakarandPundlik/makarand_unit_tests

Makarand unit tests

Commit:
Added select feature to Add TA drop down

AnuragGorkar committed:
Merge pull request #9 from MakarandPundlik/8-add-search-feature-for-select-ta-drop-down

Added select feature to Add TA drop down

Rutvik2598 committed:
Fetch students as well as TAs in the Add TA page

- Add TA only requests Users with role TA.
- This should also fetch Users with roles students.

Rutvik2598 committed:
Merge pull request #10 from MakarandPundlik/3-fetch-all-students-and-tas-in-add-ta-drop-down-1

Fetch students as well as TAs in the Add TA page

Commit:
Added permission check before rendering add TA button

AnuragGorkar committed:
Merge pull request #13 from MakarandPundlik/11-admin-only-access-to-add-ta-button

Added permission check before rendering add TA button

MakarandPundlik committed:
Removed contributor names

Commit:
Code cleanup

Rutvik2598 committed:
Instructor can also add TA other than Admin

AnuragGorkar committed:
Merge pull request #14 from MakarandPundlik/instructor_can_add_ta

Instructor can add ta
